src/routes/[slug]/+page.svelte

```diff
@@ -1,15 +1,15 @@
 <script lang="ts">
   import Header from "$lib/components/header.svelte";
   import Navigation from "$lib/components/navigation.svelte";
-  import type { Lesson } from "$lib/types";
+  import type { Page } from "$lib/types";
```

Comment:
Made this clearer - we now have a Lesson type that is exactly a Lesson as defined in course-structure, and 'Page' which has a previousSlug / nextSlug etc.